janus_core/helpers/post_process.py

```diff
@@ -189,7 +189,7 @@ def compute_vaf(
     use_velocities: bool = False,
     fft: bool = False,
     index: SliceLike = (0, None, 1),
-    filter_atoms: MaybeSequence[MaybeSequence[int]] = ((),),
+    filter_atoms: MaybeSequence[MaybeSequence[int]] = ((None),),
```

Comment:
Not `Optional`?

Reply:
They're equivalent, yes. I wrote it like this simply because it avoided further nesting of the type which is confusing enough. For consistency it probably should've been `Optional` (and now is)